### PR TITLE
test: fix several clang-tidy warnings from #5811

### PR DESCRIPTION
**Pull request by jmarantz**
*Description*: #5811 failed its clang-tidy test just due to its size, but in examining the results I found a bunch of reasonable warnings to fix, which I did thinking it might make clang-tidy CI pass. However the root cause of the CI failure was just size, and not actual errors.
*Risk Level*: low
*Testing*: //test/..., clang-tidy
*Docs Changes*: n/a
*Release Notes*: n/a

### CODE REVIEW

File: test/common/router/router_ratelimit_test.cc [hunk 1]
```diff
@@ -173,7 +173,7 @@ TEST_F(RateLimitConfiguration, NoRateLimitPolicy) {
   }
   )EOF";
 
-  SetUpTest(json);
+  setUpTest(json);
```
Comment (htuch) [inline test/common/router/router_ratelimit_test.cc]: There's a weird lexical/grammar thing going on here with `setUp`; I would write it as `setupTest`. I think you are paying homage to gtest's `SetUp`, which is also pretty weird IMHO; is it a pun on "up"?

Comment (jmarantz) [reply test/common/router/router_ratelimit_test.cc]: :)  I was just changing one character in an already-existing test-method to reflect the fact that this is a locally defined class method, not associated with gtest at all (and easy to confuse with it). So it should use Envoy's convention of a lower-cased first character.

But you're right, this looks weird, so I might as well clean this up and change it to setupTest. Any other issues?

**Comment by jmarantz** [comment]
RE why does clang-tidy not scale?  Not sure. It's processing diffs very slowly, and on a large PR there are a metric ton of diffs.  @lizan ?

**Comment by jmarantz** [comment]
And none of these were failing individually. They were just warnings.

File: test/common/router/router_ratelimit_test.cc (hunk 1 above)
Comment (htuch) [reply test/common/router/router_ratelimit_test.cc]: Nup, that's all good. I'd like to make more of these warnings act as errors though, I always prefer cleanups that don't need to ever be done again :) Thanks for doing the grungey work in this PR.

**Comment by htuch** [comment]
Please merge master to pick up https://github.com/envoyproxy/envoy/pull/5827.

**Pull request merged**